setup.py

```diff
@@ -166,6 +166,7 @@ def get_exts_for(name):
             "enum34; python_version<'3.4'",
             "funcsigs>=1.0.0; python_version=='2.7'",
             "typing; python_version<'3.5'",
+            "packaging>=17.1",
             "protobuf>=3",
             "packaging",
```

Comment:
Double dependency?

Reply:
wait, how did this end up here?

Reply:
```suggestion
```

I must be blind

Reply:
this was from a merge from a PR from jd, it wasn't a conflict, so it went unnoticed.